platform/commonUI/notification/test/suite.json

```diff
@@ -0,0 +1,3 @@
+[
+    "NotificationService"
+]
```

Comment:
I'd expect to see more tests here for coverage of other classes in platform/commonUI/notification (ideally one per script/class, though omitting constants like MessageSeverity is certainly fine)